tests/test_find_xml_file.py

```diff
@@ -0,0 +1,10 @@
+from find_xml_file.find_xml_file import find_xml_file
```

Comment:
This naming could be confusing. Can we rename these files?

Reply:
What a great suggestion, Dani!

Reply:
Since there is only one method in file, maybe we can leave this as is OR rename the file where the method lives, so its is not redundant.